### DISCUSSION TITLE
"Guest Users"

### DISCUSSION THREAD

**Discussion by MaiklT** [TandoorRecipes/recipes#1648] [Ideas]
Hi,

I have setup tandoor recipes successfully on my Synology NAS - thanks to [this article](https://github.com/TandoorRecipes/recipes/blob/develop/docs/install/synology.md). I also setup a DDNS entry, and I can reach the web site from the internet.

Now I have one question: is there a possibility for anonymous users to see the recipes? I don't want them to add, edit or delete recipes, but to read them. And then - next step - a user role that is only allowed to do the same plus add comments as well (I don't want anonymous comments)...

Thanks!
Michael

**Comment by vabene1111** [comment]
this is not yet implemented but something that might come in the future

**Comment by gloriousDan** [comment]
There is already a guest user role which allows reading and commenting. The user has to be invited via the share link same as "normal" users and the user's role can be selected when creating the invite link. Only the anonymous user part isn't yet implemented if I'm correct.

**Reply by vabene1111** [reply]
that correct, i tought he meant anonymous with guest.

**Reply by MaiklT** [reply]
OK, as far as I understood the guest role is implemented, and those users may rate and comment, but not add, edit or delete recipes.

The anonymous access is not yet implemented (users may only read recipes, no comments, no ratings). But this will be something in a furure version, correct?
